board_rules.md

```diff
@@ -0,0 +1,91 @@
+# Board rules 
+
+The description of the team workflow for the board.
+
+## Description
+
+The board consists of 5 columns:
+
+- `Common backlog`- includes all tasks that should be done.
+- `Sprint backlog` - includes the list of tasks for the current sprint.
+- `In Progress` - the list of tasks that the team works on.
+- `In Review` - the list of tasks that are under review process.
+- `Done` - the list of tasks that successfully merged to main branch.
+
+### Common backlog
+
+The process of the issue creating and adding to Common backlog.
+Every issue is placed in a common backlog when it is created.
+The issue should contain a clear title and description with definitions of done (**DoD**).
+Select necessary labels, if applicable.
+Don't forget to link the project Repository.
+
+### Sprint backlog
+
+Before sprint planning the issues place into Sprint backlog by PM.
+During the sprint planning process, the issues from the sprint backlog are estimated (labels with estimation are assigned).
+
+**Estimation** is carried out with the participation of all team members online using the **Scrum Poker** technology on the resource [planningpokeronline](https://planningpokeronline.com).
+
+Except estimation labels in progect also provided labels: 
+`high`, `medium`, `bug`, `blocker`, `enhancement`, `help wanted`:
+
+- `High` - high level of priority.
+- `Invalid`- this doesn't seem right.
+- `Documentation` - improvements or additions to documentation.
+- `Medium` - medium level of priority.
+- `Enhancement`- new feature or request.
+- `Need more discussion`- further information is requested.
+- `Bug`- something isn't working.
+- `Help wanted` - extra attention is needed.
```

Comment:
What does that mean "extra attention"?
When I should use the `Need more discussion` label and when the `Help wanted`?

Reply:
We decided to leave only one of these very similar badges: need more discussion. The badge "Help wanted" is deleted.